config.example.js

```diff
@@ -31,12 +31,12 @@ module.exports = {
   symbol: 'SYM',
   // Project description
   description: 'An awesome NFT series',
-  // Path to image to use as the main image for the collection
+  // Path to image to use as the main image for the collection. gif compatible
   // (at least 500 x 500 pixels)
   image: 'images/1.png',
-  // The URI containing JSON metadata about your contract
-  // Please follow: https://docs.opensea.io/docs/contract-level-metadata
-  contractUri: 'ipfs://...',
+  // external_link is optional. gif compatible
```

Comment:
All the other comments start with a capital letter. Please follow consistent formatting for the entire document.

Reply:
Got it and changed it